packages/server/src/instances/service.ts

```diff
@@ -153,7 +158,9 @@ export class InstanceService extends Service {
     const instance = channel.createConduit()
 
     try {
-      await instance.setup(conduitId, conduit.config, this.app)
+      await this.distributedService.using(`lock_dyn_instance_setup::${conduitId}`, async () => {
+        await instance.setup(conduitId, conduit.config, this.app)
+      })
```

Comment:
Something we could do here is set a lock that will last ~10sec and not release it. This would prevent different nodes from setting up the same conduit too fast in a row (which can lead on some channels to too many requests errors).